README.adoc

```diff
@@ -2,14 +2,18 @@
 
 An _opinionated_ Gradle plugin for Vert.x projects.
 
+image:https://img.shields.io/badge/release-0.0.6-green.svg[release, link=https://plugins.gradle.org/plugin/io.vertx.vertx-plugin]
```

Comment:
Can we have a version-agnostic link to follow versions without having to editing the `README` file?

Reply:
Yeah that was my initial thought too. But it doesn't seem to work on your GitHub Releases for some reason. Open this in your browser: https://img.shields.io/github/release/jponge/vertx-gradle-plugin.svg For me it shows "release: none"... so this was my workaround for now.

Reply:
Seems, that latest version resolves using GitHub Releases, not tags. As I know, there isn't any way get badge from Gradle Plugins directly, so for tags as versions can recommend to use: `image:https://badge.fury.io/gh/jponge%2Fvertx-gradle-plugin.svg["GitHub version", link="https://plugins.gradle.org/plugin/io.vertx.vertx-plugin"]`

Reply:
Ok, added "tag" badge to grab the latest tag version instead.